sdk/evaluation/azure-ai-evaluation/CHANGELOG.md

```diff
@@ -8,6 +8,7 @@
 
 ### Features Added
 
+- Added support for user-supplied tags in the `evaluate` function. Tags are key-value pairs that can be used for experiment tracking, A/B testing, filtering, and organizing evaluation runs. The function accepts a `tags` parameter (dictionary with up to 50 tags, where each key can have a maximum of 100 characters and each value can have a maximum of 1000 characters). Tags are automatically propagated through the evaluation pipeline and logged to both MLflow and OneDP tracking systems. A default tag `mlflow.user=azure-ai-evaluation` is automatically added unless overridden by user-supplied tags.
 - Added support for Azure OpenAI Python grader via `AzureOpenAIPythonGrader` class, which serves as a wrapper around Azure Open AI Python grader configurations. This new grader object can be supplied to the main `evaluate` method as if it were a normal callable evaluator.
```

Comment:
Since it will be supported to both, not necessay to mention it. Would prefer to include any exceptions here if any.